.travis.yml

```diff
@@ -1,2 +1,23 @@
 language: d
+dist: trusty
 sudo: false
+
+before_install:
+  - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then brew update; fi
```

Comment:
Jup, that makes a lot of sense - I wasn't sure though on whether the existing Travis config was restricting tests intentionally, so I kept it small.